### DISCUSSION TITLE
es/tasks/segment/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#18734] [Docs]
# es/tasks/segment/

Domine la segmentación de instancias utilizando YOLO11. Aprende a detectar, segmentar y contornear objetos en imágenes con guías detalladas y ejemplos.

https://docs.ultralytics.com/es/tasks/segment/

**Comment by jorgemasgomez** [comment]
Hi! When training a pre-trained segmentation model with a custom dataset, are we actually fine-tuning the model? Or is something else being done?

**Reply by glenn-jocher** [reply]
@jorgemasgomez yes, when training a pre-trained YOLO11 segmentation model on a custom dataset, you are fine-tuning the model. This process adjusts the pre-trained weights to better suit your specific dataset while leveraging the knowledge already captured during pre-training on datasets like COCO. For more details, see the [Train documentation](https://docs.ultralytics.com/modes/train/).

**Comment by UltralyticsAssistant** [comment]
👋 Hola, ¡gracias por tu interés en Ultralytics 🚀! Te recomendamos visitar la [Documentación](https://docs.ultralytics.com/es/tasks/segment/) para consultar guías detalladas y ejemplos sobre segmentación de instancias con YOLO. Aquí podrás aprender a detectar, segmentar y contornear objetos en imágenes.

Si esto es un 🐛 Reporte de Bug, por favor proporciona un [ejemplo reproducible mínimo](https://docs.ultralytics.com/help/minimum-reproducible-example/) para ayudarnos a diagnosticar el problema eficazmente.

Si esto es una ❓ Pregunta sobre entrenamiento personalizado, por favor incluye toda la información relevante, como ejemplos de imágenes del conjunto de datos, registros de entrenamiento y verifica que estás siguiendo nuestros [Consejos para los Mejores Resultados de Entrenamiento](https://docs.ultralytics.com/es/guides/model-training-tips/).

Únete a nuestra comunidad de Ultralytics en el lugar que prefieras. Para chat en tiempo real, dirígete a nuestro [Discord](https://discord.com/invite/ultralytics) 🎧. Si prefieres discusiones más detalladas, échale un vistazo a nuestro [Discourse](https://community.ultralytics.com). También puedes compartir conocimientos en nuestro [Subreddit](https://reddit.com/r/Ultralytics).

## Actualización

Actualiza al último paquete `ultralytics` incluyendo todos los [requisitos](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) en un entorno con [**Python>=3.8**](https://www.python.org/) y [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) para verificar que tu problema no haya sido resuelto ya en la última versión:

```bash
pip install -U ultralytics
```

## Entornos

YOLO puede ejecutarse en alguno de los siguientes entornos actualizados y verificados (con dependencias preinstaladas como [CUDA](https://developer.nvidia.com/cuda), [CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) y [PyTorch](https://pytorch.org/)):

- **Notebooks** con GPU gratis: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. Consulta la [Guía Rápida de GCP](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. Consulta la [Guía Rápida de AWS](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Imagen de Docker**. Consulta la [Guía Rápida de Docker](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Estado

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

Si este badge está en verde, todas las pruebas de [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) se están ejecutando correctamente. Las pruebas de CI verifican la operación correcta de todos los [Modos](https://docs.ultralytics.com/es/modes/) y [Tareas](https://docs.ultralytics.com/es/tasks/) de YOLO en macOS, Windows y Ubuntu cada 24 horas y en cada commit.

Este es un mensaje automatizado, pero un ingeniero de Ultralytics también te ayudará pronto 😊.
